### PR TITLE
Task/MYR-287: Tidy up "My Wallet" in my profile

### PR DESCRIPTION
**Pull request by ical10**

### CODE REVIEW

**Review by rei-gun — CHANGES_REQUESTED** [review]

File: src/components/profile/myWallet/tipping/currencyDetail.component.tsx [hunk 1]
```diff
@@ -98,41 +101,86 @@ const TableCell = withStyles({
   },
 })(MuiTableCell);
 
-export const CurrencyDetails = () => {
+interface CurrencyDetailComponentProps {
+  userTransactionDetails: UserTransactionDetail[];
+  balanceDetail: BalanceDetail;
+}
+
+interface TokenDetail {
```
[the remark this answers is not in the document]
Comment (ical10) [reply src/components/profile/myWallet/tipping/currencyDetail.component.tsx:109]: siap

File: src/reducers/user/actions.ts [hunk 2]
```diff
@@ -28,6 +28,11 @@ export interface FetchUserToken extends Action {
   payload: Token[];
 }
 
+export interface FetchUserTransactionDetails extends Action {
```
[the remark this answers is not in the document]
Comment (ical10) [reply src/reducers/user/actions.ts:31]: let me check and I'll make sure that they're consistent

File: src/components/profile/myWallet/tipping/currencyDetail.component.tsx [hunk 3]
```diff
@@ -98,41 +101,86 @@ const TableCell = withStyles({
   },
 })(MuiTableCell);
 
-export const CurrencyDetails = () => {
+interface CurrencyDetailComponentProps {
```
[the remark this answers is not in the document]
Comment (ical10) [reply src/components/profile/myWallet/tipping/currencyDetail.component.tsx:104]: siap boss

File: src/reducers/user/actions.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (ical10) [reply src/reducers/user/actions.ts:31]: yes, "get" for APIs and "fetch" for actions.

**Review by rei-gun — APPROVED** [review]

**Pull request merged**